src/java/org/apache/cassandra/service/accord/interop/AccordInteropExecution.java

```diff
@@ -290,6 +291,66 @@ private AsyncChain<Data> readChains()
                 }));
             });
         });
+        return results;
+    }
+
+    private List<AsyncChain<Data>> rangeReadChains(long nowInSeconds, Dispatcher.RequestTime requestTime)
+    {
+        TxnRangeRead read = (TxnRangeRead) txn.read();
+        Seekables<?, ?> keys = txn.read().keys();
```

Comment:
nit: `ranges`?

Reply:
This is all going away in the next patch, where I unify `TxnRangeRead` into `TxnRead` and `TxnNamedRead` (`TxnKeyRead` also goes away). I'll rename it.

Reply:
Also if you don't like keys standing in for ranges you are going to see it a *lot* and it's already like that in Accord. If personally don't think it's worth repeating `keysOrRanges` if something is a `Seekable/Seekables` just because it's implied in the choice of type.

We can make an effort rename it everywhere, but it goes stale pretty quickly as things get converted to handle `Seekable` instead of something more specific.